core/bentley/src/Logger.ts

```diff
@@ -6,10 +6,69 @@
  * @module Logging
  */
 
+import type { ContextAPI, SpanAttributes, SpanAttributeValue, SpanContext, SpanOptions, TraceAPI, Tracer } from "@opentelemetry/api";
 import { BentleyError, IModelStatus, LoggingMetaData } from "./BentleyError";
 import { BentleyLoggerCategory } from "./BentleyLoggerCategory";
 import { IDisposable } from "./Disposable";
 
+// re-export so that consumers can construct full SpanOptions object without external dependencies
+export { SpanKind } from "@opentelemetry/api";
```

Comment:
This still introduces a dependency on the `@opentelemetry/api` package...

Would we be better off just re-implementing it?

Reply:
well... let's hope it doesn't change in the future :D